nx-X11/programs/Xserver/xkb/xkbUtils.c

```diff
@@ -51,12 +51,8 @@ int	XkbDisableLockActions = 0;
 /***====================================================================***/
 
 #ifndef RETURN_SHOULD_REPEAT
-#if (defined(__osf__) && defined(__alpha))
-#define RETURN_SHOULD_REPEAT 1
-#else
 #define	RETURN_SHOULD_REPEAT 0
 #endif
-#endif
 
 /***====================================================================***/
 
```

Comment:
Not for this file, but `nx-X11/config/cf/Imake.tmpl`: `OSF1Architecture` left-overs.